packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $anchorScroll,
+    $location,
+    $translate,
+    Alerter,
+    datacenterBackupService,
+  ) {
+    this.$anchorScroll = $anchorScroll;
+    this.$location = $location;
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.backup = {};
+    this.alerter = Alerter;
```

Comment:
alerter is defined twice https://github.com/ovh/manager/pull/2320/files#diff-f1f66957f52691f22d275f3fcec7a002R15

Reply:
Additional definition removed. Thanks